src/components/Review.jsx

```diff
@@ -0,0 +1,67 @@
+import { useState, useEffect } from "react";
+import { LoadingOutlined } from "@ant-design/icons";
+import * as api from "../api.js";
+import { useParams, Link } from "react-router-dom";
+import { Breadcrumb, Layout, Row, Col } from "antd";
+
+const { Content } = Layout;
+
+export default function Review({ isLoading, setIsLoading }) {
+  const { review_id } = useParams();
+  const [review, setReview] = useState({});
+  useEffect(() => {
+    setIsLoading(true);
+    api.getReview(review_id).then((data) => {
+      setReview(data.review);
+      setIsLoading(false);
+    });
+  }, [review_id]);
+
+  if (isLoading) {
+    return (
+      <>
+        <br />
+        <LoadingOutlined style={{ fontSize: 96 }} spin />
+      </>
+    );
+  }
+  const {
```

Comment:
rather than create an uncontrolled state variable, you can just destructure reviews in your useState statement